gslib/commands/logging.py

```diff
@@ -186,10 +186,10 @@ class LoggingCommand(Command):
           'get':
               GcloudStorageMap(
                   gcloud_command=[
-                      'storage', 'buckets', 'list',
-                      '--format="gsutiljson[key=logging_config,empty=\' has '
+                      'storage', 'buckets', 'describe',
```

Comment:
So, logging was earlier printing details for multiple buckets, but we intend to print it only for one. This is why we are now moving to `describe`. Is the understanding correct ?  
Also, have we manually tested this ?

Reply:
logging was never printing details for multiple buckets. `gsutil logging get` command shows logging configuration for only one bucket. That's the reason to use describe command as other commands are using. 

I sent you the manual testing results internally!

Reply:
Discussing internally.